packages/cozy-sharing/README.md

```diff
@@ -2,12 +2,26 @@
 
 Open the playgrounds in cozy-libs and run `yarn start`
 
-# How to use the lib 
+# How to use the lib
 
-In your app, you have to : 
+In your app, you have to :
 
-- import the Provider: ```import SharingProvider, { ShareButton, ShareModal } from 'cozy-sharing'```
-- import the sylesheet.css: ```import 'cozy-sharing/dist/stylesheet.css'```
+- import the Provider: `import SharingProvider, { ShareButton, ShareModal } from 'cozy-sharing'`
+- import the sylesheet.css: `import 'cozy-sharing/dist/stylesheet.css'`
```

Comment:
```suggestion
- import the stylesheet.css: `import 'cozy-sharing/dist/stylesheet.css'`
```

In the example, it is not clear why we need CSS, could there be an example showing
a component needing CSS ?